stable/redis-ha/values.yaml

```diff
@@ -66,9 +60,13 @@ persistentVolume:
   ##
   subPath: ""
 ## Redis image version
-redis_image: quay.io/smile/redis:4.0.2
+redis_image: quay.io/ianmaddox/redis-sentinel:redis-4.0.6r0
 ## replicas number for each component
 replicas:
-  master: 1
-  slave: 1
-  sentinel: 3
+  servers: 3
+  sentinels: 3
+appliance:
+  # Set vpcMode to true to automatically open the master and slave load balancers to access from within the VPC
+  vpcMode: false
+  annotations:
```

Comment:
This applies to the service for a particular component. Can you add the configuration in that section of the values file?